src/Attribute/Types/MysqlTypes.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace CoolBeans\Attribute\Types;
+
+class MysqlTypes
```

Comment:
use Nette\StaticClass;

Include all possible mysql types - or at least most of them.